src/main/c/Jep/pyembed.c

```diff
@@ -206,6 +247,8 @@ void pyembed_preinit(JNIEnv *env,
                      jint hashRandomizationFlag,
                      jstring pythonHome)
 {
+    // TODO: Most of these flags can't be implemented with the Stable ABI :(
+#ifndef Py_LIMITED_API
```

Comment:
It seems like this is more proof we need more than what the Stable API can provide.

Python's CAPI interface is definitely a leaky abstraction, but we need the whole thing, warts and all.
We want to be able to control execution just as much as the `cpython` binary can......

For example the `Py_DontWriteBytecodeFlag` could be pretty important in restricted environments.....